### PR TITLE
Walkthrough notebook (and Embeddings script)

### PR DESCRIPTION
**Pull request by maartenschermer**
Walkthrough notebook + sample data files.
Accidentally tagged onto another branch (`embeddings-script`), hopefully not an issue!

### CODE REVIEW

File: examples/data/GPCPCs.txt
Comment (qubixes) [inline examples/data/GPCPCs.txt:1]: Did you check with Benjamin if it's okay to share these files?

They are reasonably big (by git standards), maybe it makes sense to sample them?

File: examples/walkthrough.ipynb
Comment (qubixes) [inline examples/walkthrough.ipynb:1]: Looks like a really nice and complete walkthrough, thanks! Would it be possible to make some of the cells markdown to add some captions/section to the file?

**Review by qubixes — CHANGES_REQUESTED** [review]
Looks really great, thanks! Just some minor questions from my side.

**Review by ShNadi — COMMENTED** [review]
Nice and clear walkthrough, thanks!
Does it make sense to include greek, stopwords, and symbol files for the preprocessing in the data folder?

**Comment by maartenschermer** [comment]
@ShNadi I don't think we actually use them in the final version.
Nb, I've asked Benjamin about the other files, and he thinks it's a good idea, but he wants to check with his supervisor.

File: examples/walkthrough.ipynb
Comment (qubixes) [reply examples/walkthrough.ipynb:1]: Okay, if it is a big problem, don't worry too much about it. If I can help with anything, let me know.

**Comment by qubixes** [comment]
> @ShNadi I don't think we actually use them in the final version. Nb, I've asked Benjamin about the other files, and he thinks it's a good idea, but he wants to check with his supervisor.

Aren't they used in the `OldPreprocessor` class? At least as far as I can tell they are used, but I could be wrong.

**Comment by maartenschermer** [comment]
> Aren't they used in the `OldPreprocessor` class? At least as far as I can tell they are used, but I could be wrong.

That's true, but `OldPreprocessor` is only used in the hyperopt tuning, so maybe we should put the stopword and symbol files there?

**Comment by qubixes** [comment]
> > Aren't they used in the `OldPreprocessor` class? At least as far as I can tell they are used, but I could be wrong.
> 
> That's true, but `OldPreprocessor` is only used in the hyperopt tuning, so maybe we should put the stopword and symbol files there?

Well, the best results are with the OldPreprocessor according to the hyperopt notebook. @ShNadi is now trying to do the run with the old preprocessor..

**Comment by maartenschermer** [comment]
Also, Benjamin talked to his supervisor, and they are ok with the sample data files.

**Comment by qubixes** [comment]
Looks great. Just a minor point: The tf-idf model should use the new preprocessor, not the old one. It's only the CountVec model that uses the old preprocessing method.

**Comment by qubixes** [comment]
> Ah, ok, I thought I also saw better values with the old one for the Tf-Idf.

I checked now, and you're indeed right. Given that we have run it with  the new preprocessor, might be better to just keep it like that. The difference is well within error margins anyway (not so much for the countvec model!).

**Pull request merged**